foundation/settings.py

```diff
@@ -105,7 +121,7 @@ def _parse_email_list(varname):
 
     # 3rd-party important
     'reversion',
-    's3_folder_storage',
+    # only for S3 's3_folder_storage',
```

Comment:
So not needed anymore?

Reply:
Changed [here](https://github.com/okfn/website/commit/890a9702f58465a79b4bd428b14f8c8841429367)